src/main/java/com/segment/analytics/android/integrations/firebase/FirebaseIntegration.java

```diff
@@ -80,7 +80,7 @@ public String key() {
     EVENT_MAPPER.put("Product Added to Wishlist", Event.ADD_TO_WISHLIST);
     EVENT_MAPPER.put("Product Shared", Event.SHARE);
     EVENT_MAPPER.put("Product Clicked", Event.SELECT_CONTENT);
-    EVENT_MAPPER.put("Product Searched", Event.SEARCH);
+    EVENT_MAPPER.put("Products Searched", Event.SEARCH);
```

Comment:
do we want to add a new mapper for `Products Searched`, rather than replacing the previous one? im guessing customers might be using `Product Searched` currently and this change would break that flow

Reply:
nvm, seems like the correct one to use is `Products Searched` (ref: https://github.com/segment-integrations/analytics-ios-integration-firebase/blob/master/Segment-Firebase/Classes/SEGFirebaseIntegration.m#L104), so its ok to replace here